pom.xml

```diff
@@ -27,6 +27,7 @@
     <module>cdap-ranger</module>
     <module>cdap-sentry</module>
     <module>cdap-security-extensions-dist</module>
+    <module>cdap-authorization-ldap-role</module>
```

Comment:
Please change this to reflect the new module name (`cdap-ldap-role`), thanks!

Reply:
done